File: CS/MySolution.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.ExpressApp.EF;


namespace MySolution.Module {
    public sealed partial class MySolutionModule : ModuleBase {
        public MySolutionModule() {
            InitializeComponent();
            ExportedTypeHelpers.AddExportedTypeHelper(typeof(EFExportedTypeHelperCF).FullName);
       }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            return ModuleUpdater.EmptyModuleUpdaters;
        }
    }
}

File: VB/MySolution.Module/Module.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.ExpressApp.EF
Imports System.Data.Entity


Namespace MySolution.Module
	Public NotInheritable Partial Class MySolutionModule
		Inherits ModuleBase

		Public Sub New()
			InitializeComponent()
			Database.SetInitializer(Of MyDbContext)(New DropCreateDatabaseIfModelChanges(Of MyDbContext)())
		End Sub
		Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
			Dim updater As ModuleUpdater = New Updater(objectSpace, versionFromDB)
			Return New ModuleUpdater() { updater }
		End Function
	End Class
End Namespace
